csharp/src/Ice/ThreadPool.cs

```diff
@@ -344,27 +336,10 @@ public void Finish(EventHandler handler)
 
         public void DispatchFromThisThread(System.Action call, Ice.Connection? con)
         {
-            if (_dispatcher != null)
-            {
-                try
-                {
-                    _dispatcher(call, con);
-                }
-                catch (System.Exception ex)
-                {
-                    if ((_communicator.GetPropertyAsInt("Ice.Warn.Dispatch") ?? 1) > 1)
-                    {
-                        _communicator.Logger.Warning("dispatch exception:\n" + ex);
-                    }
-                }
-            }
-            else
-            {
-                call();
-            }
+            call();
```

Comment:
Maybe we should remove the con parameter here and bellow until we add a replacement, I assume this adds a warning for the unused param, also mark it as TODO,

Reply:
Removed.